packages/core/src/shared/resourcefetcher/httpResourceFetcher.ts

```
@@ -82,7 +86,10 @@ export class HttpResourceFetcher implements ResourceFetcher<Response> {
                 `Error downloading ${this.logText()}: %s`,
                 error.message ?? error.code ?? error.response.statusText ?? error.response.status
             )
-            return undefined
+            if (this.params.swallowError) {
```

previously this is swallowing all errors from fetch. With this change we can detect and catch the user cancellation errors.